lib/components/avatar.css

```diff
@@ -0,0 +1,63 @@
+.av {
+    border-radius: 100%;
+    height: 4.5rem;
+    width: 4.5rem;
+}
+
+.av-sm,
+.av-md, 
+.av-lg, 
+.av-xl {
+    border-radius: 100%;
+}
+
+.av-sm {
+    height: 3.5rem;
+    width: 3.5rem;
+}
+
+.av-md {
+    height: 4rem;
+    width: 4rem;
+}
+
+.av-lg {
+    height: 4.5rem;
+    width: 4.5rem;
+}
```

Comment:
use comma selector as the properties are common with `.av` class

Reply:
Done.
Changed properties of .av class to other value.
Thanks